app/templates/webchat-form.html

```diff
@@ -1,12 +1,15 @@
 {% if display_region == "cy" %}
     {% extends "base-webchat-cy.html" %}
     {% set form_action = url('WebChatCY:post') %}
+    {% set help_url = '/help/' %}
```

Comment:
If these really are identical, they can be factored out of the conditional.

Reply:
They won't be different, but still awaiting the url value for NI and Welsh